.final_builds/packages/golang-1.10-windows/index.yml

```diff
@@ -0,0 +1,6 @@
+builds:
```

Comment:
Any particular reason you use Go 1.10?

Reply:
I think when we started on our work CFCR was still on 1.10 as well, we'll update to match